src/heliaServer.ts

```diff
@@ -112,6 +104,9 @@ export class HeliaServer {
       this.log('Requesting content from helia:', request.url)
       let type: string | undefined
       const { address, namespace } = this.parseHostParts(request.hostname)
+      if (address === '' || namespace === '') {
+        return await reply.code(200).send('try /ipfs/<cid> or /ipns/<name>')
```

Comment:
they "try /blah" and "blah" should be done at the route level. 

```suggestion
        return await reply.code(500).send(`Error parsing address and namespace for ${request.url}`)
```